hash_version.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/bash
+#!/bin/sh
 
-GIT_HASH=`sh ./calc_hash.sh`;
+GIT_HASH=$(sh ./calc_hash.sh)
 
+# NOTICE:
+# don't use sed -i, because it's not portable between Linux, OS X and some BSDs
 rm -f ver
-sed -b "s/NO_GIT_COMMIT_HASH_DEFINED/$GIT_HASH$1/" renderdoc/data/resource.h > ver && mv ver renderdoc/data/resource.h
-sed -b "s/NO_GIT_COMMIT_HASH_DEFINED/$GIT_HASH$1/" renderdocui/Properties/AssemblyInfo.cs > ver && mv ver renderdocui/Properties/AssemblyInfo.cs
+sed "s/NO_GIT_COMMIT_HASH_DEFINED/$GIT_HASH$1/" renderdoc/data/resource.h > ver && mv ver renderdoc/data/resource.h
```

Comment:
Hmm, we could pass info about OS to this script or use `if [ "$(uname -s | grep -c -E "CYGWIN*|MINGW*|MSYS*)" -gt 0 ]` to detect Windows and then do `sed 's/$/\r/g'`. What do you think?